monai/transforms/utils.py

```diff
@@ -2051,5 +2058,136 @@ def has_status_keys(data: torch.Tensor, status_key: Any, default_message: str =
     return True, None
 
 
+def distance_transform_edt(
+    img: NdarrayOrTensor,
+    sampling: None | float | list[float] = None,
+    return_distances: bool = True,
+    return_indices: bool = False,
+    distances: NdarrayOrTensor | None = None,
+    indices: NdarrayOrTensor | None = None,
+    block_params: tuple[int, int, int] | None = None,
```

Comment:
I think we should make block_params and float64_distances keyword only arguments just like the cucim implementation does.

Reply:
Ah sweet, I did not yet know that functionality. Readded :)